Panic when creating a wallet dir for the first time

To reproduce: try to create a new wallet on a new data dir.

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0x100a87dd7]

goroutine 209 [running]:
github.com/crypto-power/cryptopower/libwallet/internal/loader.(*Loader).CreateDirPath(0xc0035207c0, {0x10174c5c6, 0x1}, {0x101712169, 0x9}, {0x101709717?, 0xc0033d9410?})
        /Users/philemon/Desktop/osp/cryptopower/libwallet/internal/loader/config.go:116 +0x217
github.com/crypto-power/cryptopower/libwallet/internal/loader/ltc.(*ltcLoader).getWalletLoader(0xc005a0d680, {0x10174c5c6?, 0x8?}, 0x0?)
        /Users/philemon/Desktop/osp/cryptopower/libwallet/internal/loader/ltc/loader.go:77 +0x4b
github.com/crypto-power/cryptopower/libwallet/internal/loader/ltc.(*ltcLoader).CreateNewWallet(0xc005a0d680, {0x102aa6508?, 0x1?}, 0xc0000a29c0)
        /Users/philemon/Desktop/osp/cryptopower/libwallet/internal/loader/ltc/loader.go:118 +0xf1
github.com/crypto-power/cryptopower/libwallet/assets/wallet.(*Wallet).createWallet(0xc003012120, {0x102aa6508, 0x1}, {0xc000f6e480, 0x11f})
        /Users/philemon/Desktop/osp/cryptopower/libwallet/assets/wallet/wallet_shared.go:374 +0x2b9
github.com/crypto-power/cryptopower/libwallet/assets/wallet.CreateNewWallet.func1()
        /Users/philemon/Desktop/osp/cryptopower/libwallet/assets/wallet/wallet_shared.go:350 +0x65
github.com/crypto-power/cryptopower/libwallet/assets/wallet.(*Wallet).saveNewWallet.func1({0x101e18cd0, 0xc003524380})
        /Users/philemon/Desktop/osp/cryptopower/libwallet/assets/wallet/wallet_shared.go:536 +0x1fd
github.com/crypto-power/cryptopower/libwallet/assets/wallet.(*Wallet).batchDbTransaction(0xc003012120?, 0xc0000c0d68)
        /Users/philemon/Desktop/osp/cryptopower/libwallet/assets/wallet/wallet_utils.go:112 +0xb6
github.com/crypto-power/cryptopower/libwallet/assets/wallet.(*Wallet).saveNewWallet(0xc003012120, 0xc0000c0e00)
        /Users/philemon/Desktop/osp/cryptopower/libwallet/assets/wallet/wallet_shared.go:504 +0x72
github.com/crypto-power/cryptopower/libwallet/assets/wallet.CreateNewWallet(0xc0000c0ed8, {0x101e0a138?, 0xc005a0d680}, 0xc00010ebe0, {0x101709717, 0x3})
        /Users/philemon/Desktop/osp/cryptopower/libwallet/assets/wallet/wallet_shared.go:345 +0x2e5
github.com/crypto-power/cryptopower/libwallet/assets/ltc.CreateNewWallet(0x1?, 0xc00010ebe0)
        /Users/philemon/Desktop/osp/cryptopower/libwallet/assets/ltc/wallet.go:101 +0x79
github.com/crypto-power/cryptopower/libwallet.(*AssetsManager).CreateNewLTCWallet(0xc002f55f40, {0xc0044282f0?, 0xc003085a00?}, {0x102aa6508?, 0xc003085a00?}, 0x64f00?)
        /Users/philemon/Desktop/osp/cryptopower/libwallet/ltc.go:31 +0x59
github.com/crypto-power/cryptopower/ui/page/components.(*CreateWallet).HandleUserInteractions.func1()
        /Users/philemon/Desktop/osp/cryptopower/ui/page/components/wallet_setup_page.go:461 +0x4cd
created by github.com/crypto-power/cryptopower/ui/page/components.(*CreateWallet).HandleUserInteractions in goroutine 26
        /Users/philemon/Desktop/osp/cryptopower/ui/page/components/wallet_setup_page.go:427 +0x3c6
exit status 2

```

--- a/libwallet/internal/loader/config.go
+++ b/libwallet/internal/loader/config.go
@@ -95,23 +95,14 @@ func (l *Loader) CreateDirPath(WalletID, walletDbName string, assetType utils.As
 
 	folderPath := filepath.Join(l.DbDirPath, assetType.ToStringLower(), WalletID)
 	// Ensure that the network directory exists.
-	file, err := os.Stat(folderPath)
-	if err != nil {
-		if os.IsNotExist(err) {
-			// error expected thus now attempt data directory creation
-			if err = os.MkdirAll(folderPath, utils.UserFilePerm); err != nil {
-				return "", err
-			}
-		} else {
-			// if os.IsNotExist(err) returned false means that unexpected error
-			// was returned thus exit the function with that error.
+	if file, err := os.Stat(folderPath); err != nil {
+		if !os.IsNotExist(err) {
 			return "", err
 		}
-	}
-
-	// No error was returned. Else used here to maintain the scope of
-	// file variable just with the if-else statement.
-	if !file.IsDir() {
+		if err = os.MkdirAll(folderPath, utils.UserFilePerm); err != nil {
+			return "", err
+		}
+	} else if !file.IsDir() {
 		return "", errors.Errorf("%q is not a directory", folderPath)
 	}
 

